package.json

```diff
@@ -19,6 +14,12 @@
     "prepublish": "yarn build:library",
     "start": "react-scripts start"
   },
+  "main": "lib/index.js",
+  "files": [
+    "package.json",
+    "README.md",
+    "lib"
+  ],
```

Comment:
yeah it's a nice one :) https://docs.npmjs.com/files/package.json#files
The package.json and README are included by default with this but I listed them here to be explicit